stories/paragraph.stories.js

```diff
@@ -5,8 +5,14 @@ import { Paragraph } from '../src';
 
 storiesOf('Paragraph', module).add('default', () => (
   <Fragment>
-    <Paragraph.p variant="ui">Paragraph (UI)</Paragraph.p>
-    <Paragraph.p variant="longForm">Paragraph (Long Form)</Paragraph.p>
-    <Paragraph.p variant="tiny">Paragraph (Tiny)</Paragraph.p>
+    <Paragraph element="p" variant="ui">
+      Paragraph (UI)
+    </Paragraph>
+    <Paragraph element="p" variant="longForm">
```

Comment:
Since `Text` has a defaultProp for element as `p` do we need to specify it here?